src/WebJobs.Script/DependencyInjection/InstanceServices/ScriptInstanceServicesProvider.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.WebJobs.Script.DependencyInjection
+{
+    internal class ScriptInstanceServicesProvider : IInstanceServicesProvider, IDisposable
```

Comment:
How hard would it be to add a regression test for this issue?

Reply:
Planning on adding that prior to going into ‘dev’ as discussed offline. Main goal at the moment is getting the branch to a clean test/functional state and unblocking @kshyju ’s validations